diff --git a/PKGBUILD b/PKGBUILD
index 1a71ace..5ad3735 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -48,6 +48,13 @@ sha256sums=(
   'ced3b935d4f4a04e98da215862de6b6ab4282669fd20ff0b039c913b2f322dec'
 )
 
+prepare() {
+  cd ${srcdir}
+  rm NerdFonts/OpenDyslexic*
+  font-line percent 1 NerdFonts/*.ttf
+  font-line percent 1 NerdFonts/*.otf
+}
+
 package() {
   local -r libdir="${pkgdir}/usr/lib/${pkgname}"
   local -r licensedir="${pkgdir}/usr/share/licenses/${pkgname}"
@@ -58,13 +65,13 @@ package() {
   install -d -m 755 "${otfdir}"
   while read -r font; do
     install -m 644 "${font}" "${otfdir}"
-  done < <(find NerdFonts -name "*.otf")
+  done < <(find NerdFonts -name "*-linegap1.otf")
 
   # ttf:
   install -d -m 755 "${ttfdir}"
   while read -r font; do
     install -m 644 "${font}" "${ttfdir}"
-  done < <(find NerdFonts -name "*.ttf")
+  done < <(find NerdFonts -name "*-linegap1.ttf")
 
   # Scripts:
   install -d -m 755 "${libdir}"
