diff --git a/config.def.h b/config.def.h
index 6f05dce..c17b56c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -95,31 +95,28 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
+	"#22262e", /* color0 */
+	"#e27878", /* color1 */
+	"#b4be82", /* color2 */
+	"#e2a478", /* color3 */
+	"#84a0c6", /* color4 */
+	"#a093c7", /* color5 */
+	"#89b8c2", /* color6 */
+	"#c6c8d1", /* color7 */
+	"#6b7089", /* color8 */
+	"#e98989", /* color9 */
+	"#c0ca8e", /* color10 */
+	"#e9b189", /* color11 */
+	"#91acd1", /* color12 */
+	"#ada0d3", /* color13 */
+	"#95c4ce", /* color14 */
+	"#d2d4de", /* color15 */
 	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	// more colors can be added after 255 to use with DefaultXX
+	"#c6c8d1", /* 256 -> cursor */
+	"#161821", /* 257 -> rev cursor*/
+	"#161821", /* 258 -> bg */
+	"#c6c8d1", /* 259 -> fg */
 };
 
 
@@ -127,8 +124,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
+unsigned int defaultfg = 259;
+unsigned int defaultbg = 258;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
