diff --git a/config.def.h b/config.def.h
index 293e00c..06b79c4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -16,7 +16,7 @@ static int borderpx = 2;
  * 4: value of shell in /etc/passwd
  * 5: value of shell in config.h
  */
-static char *shell = "/bin/sh";
+static char *shell = "/bin/zsh";
 char *utmp = NULL;
 /* scroll program: to enable use a string like "scroll" */
 char *scroll = NULL;
@@ -34,7 +34,7 @@ static float chscale = 1.0;
  *
  * More advanced example: L" `'\"()[]{}"
  */
-wchar_t *worddelimiters = L" ";
+wchar_t *worddelimiters = L" '`\"()[]{}<>|";

 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
@@ -87,7 +87,7 @@ char *termname = "st-256color";
  *
  *	stty tabs
  */
-unsigned int tabspaces = 8;
+unsigned int tabspaces = 4;

 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
@@ -141,8 +141,8 @@ static unsigned int cursorshape = 2;
  * Default columns and rows numbers
  */

-static unsigned int cols = 80;
-static unsigned int rows = 24;
+static unsigned int cols = 125;
+static unsigned int rows = 40;

 /*
  * Default colour and shape of the mouse cursor
